applications/konami/src/konami_code_fsm.erl

```diff
@@ -486,26 +486,40 @@ arm_bleg(#state{digit_timeout=Timeout}=State) ->
                 ,b_leg_armed='true'
                }.
 
+-spec maybe_fast_rearm(ne_binary(), ne_binary(), binary()) -> binary().
+-spec maybe_fast_rearm(ne_binary(), ne_binary(), binary(), boolean()) -> binary().
+
+maybe_fast_rearm(DTMF, BindingDigit, Collected) ->
+    maybe_fast_rearm(DTMF, BindingDigit, Collected
+        ,kapps_config:get_is_true(<<"metaflows">>, <<"use_fast_rearm">>, 'false')).
```

Comment:
OK. I see this is more konami-scoped config. I will need to change the docs too in this case.